x-pack/plugins/enterprise_search/server/utils/create_pipeline_definitions.ts

```diff
@@ -5,12 +5,17 @@
  * 2.0.
  */
 
+import { IngestPipeline } from '@elastic/elasticsearch/lib/api/types';
 import { ElasticsearchClient } from '@kbn/core/server';
 
 export interface CreatedPipelines {
   created: string[];
 }
 
+export interface ESPipeline extends IngestPipeline {
```

Comment:
I've never liked the shorted name "ES" since it could mean EnterpiseSearch or Elasticsearch. What do we really want this to mean? `EnterpriseSearchPipeline`, `ElasicsearchPipeline`, `MlInferencePipeline`, something else?